packages/plugin/src/classes/helpers/classes.js

```diff
@@ -222,37 +222,58 @@ export function convertClassToUI5Extend(
       // @transformControllerExtension marker, because it is not a type assignment. In the resulting code, the
       // "ControllerExtension.use(...)" part should be removed and the content of the brackets should be assigned
       // directly to the member property.
-      if (t.isCallExpression(member.value)) {
-        const callee = member.value.callee;
-        if (
-          t.isMemberExpression(callee) &&
-          t.isIdentifier(callee.object) &&
-          t.isIdentifier(callee.property) &&
-          callee.property.name === "use" // we are looking for "ControllerExtension.use(...)"
-        ) {
-          const importDeclaration = getImportDeclaration(
-            memberPath?.hub?.file?.opts?.filename,
-            callee?.object?.name // usually, but not necessarily always: "ControllerExtension"...
-          );
-          // ...hence we rather look at the imported module name to be sure
+      if (
+        t.isCallExpression(member.value) ||
+        t.isSequenceExpression(member.value)
+      ) {
+        let callExpression = member.value;
+
+        // code instrumentation sometimes wraps it like:
+        // this.routing = (cov_1uvvg22e7l().s[5]++, ControllerExtension.use(Routing.override({ … })));
+        if (t.isSequenceExpression(member.value)) {
+          // iterate through the expressions in the sequence
+          for (const expr of member.value.expressions) {
+            if (t.isCallExpression(expr)) {
+              callExpression = expr;
+              break;
+            }
+          }
+        }
+
```

Comment:
Souldn't you consider the last one ? I mean (1,2,3) returns 3

Reply:
As Arnaud said, it should always just consider the last entry of a sequence - independent from what's in.

`const expr = member.value.expressions.pop();`

Reply:
When we keep the entire sequence, as correctly suggested by Peter, then this is not a problem anymore, as we don't pick anything.